seyik/app.js

```diff
@@ -25,17 +25,33 @@ appDataSource
     appDataSource.destroy();
   });
 
-  
 const app = express();
 
 app.use(express.json());
-app.use(cors());
+app.use(cors()); // 모든 통신이 cors 메소드를 통과해야지만 올바르게 동작되어 리스폰스를 쏴줄수 있다.
 app.use(morgan("dev"));
 
 app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.post("/users/signup", async (req, res, next) => {
+  const { name, email, profileImage, password } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO users(
+          name, 
+          email,
+          profile_image,
+          password
+      ) VALUES (?, ?, ?, ?);
+      `,
+    [name, email, profileImage, password]
+  );
+
+  res.status(201).json({ message: "userCreated" });
+});
+
 app.listen(3000, function () {
   console.log("server listening on port 3000");
 });
```

Comment:
```suggestion
app.listen(PORT, function () {
  console.log(`server listening on port ${PORT}`);
});
```
1. api server에 대한 port number는 기존과 같이 하드코딩 하는것이 아닌, `.env`를 통해 환경변수로서 관리하고 이를 활용하여 작성될 수 있도록 해주세요.
2. 또한 console.log()를 통해 해당 서버가 어떠한 port number를 통해 열려있는지도 확인할 때, 자바스크립트 템플릿 리터럴(`${변수 이름}`)을 통해 작성될 수 있게 해주세요. 이를 위해서는 작은 따옴표나 큰 따옴표가 아닌 백틱(`)으로 감싸져야 합니다.

```javascript
. . .
const PORT = process.env.PORT
. . .
```